packages/shared/lib/migration.ts

```diff
@@ -122,7 +122,7 @@ export const migration = writable<MigrationState>({
     data: writable<MigrationData>({
         lastCheckedAddressIndex: 0,
         balance: 0,
-        inputs: []
+        inputs: [],
```

Comment:
Don't you need spentAddresses also here?

Reply:
No, it is only needed for checking if we need to generate the next batch of addresses. So, I decided to not store it in svelte store. Also kept it optional in the types.